packages/tests/integration/src/relations/ManyToManyCollection.test.ts

```diff
@@ -231,8 +242,7 @@ describe("ManyToManyCollection", () => {
 
     book.tags.add(tag);
     expect(tag.books.get).toContain(book);
-    expect((await book.tags.load()).length).toBe(1);
```

Comment:
For some reason this test was asserting against the prior buggy behavior; after the add, `2` is the right length here. :shrug: